nomad/alloc_endpoint_test.go

```diff
@@ -1284,8 +1284,9 @@ func TestAllocEndpoint_List_AllNamespaces_ACL_OSS(t *testing.T) {
 		{
 			Label:     "all namespaces with insufficient token",
 			Namespace: "*",
-			Allocs:    []*structs.Allocation{},
 			Token:     ns1tokenInsufficient.SecretID,
+			Error:     true,
+			Message:   structs.ErrPermissionDenied.Error(),
```

Comment:
This was the only test the failed after the refactoring, but I think returning a `403` is the correct result in this case? The test token doesn't have the `read-job` capability.